### PR TITLE
Allow to filter the automatically selected slack bus on countries

### PR DESCRIPTION
**Pull request by EtienneLt**
Signed-off-by: Etienne LESOT <etienne.lesot@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
feature

**What is the new behavior (if this is a feature change)?**
allow to filter the automatically selected slack bus on countries (Most Meshed, First and Largest Generator modes)

### CODE REVIEW

File: src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java [hunk 1]
```diff
@@ -170,6 +171,8 @@ public class OpenLoadFlowParameters extends AbstractExtension<LoadFlowParameters
 
     private static final String ALWAYS_UPDATE_NETWORK_PARAM_NAME = "alwaysUpdateNetwork";
 
+    public static final String COUNTRIES_TO_FILTER_SLACK_BUS_PARAM_NAME = "countriesToFilterSlackBus";
```
Comment (annetill) [inline src/main/java/com/powsybl/openloadflow/OpenLoadFlowParameters.java:174]: @annetill maybe something close to `countriesForSlackBusSelection`.

File: src/test/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImplTest.java [hunk 2]
```diff
@@ -118,15 +118,13 @@ void networkWith3wtTest() {
         network = ThreeWindingsTransformerNetworkFactory.create();
         ThreeWindingsTransformer transformer = network.getThreeWindingsTransformer("3WT");
         assertNotNull(transformer);
-        VoltageLevel voltageLevelLeg1 = transformer.getLeg1().getTerminal().getVoltageLevel();
```
[the remark this answers is not in the document]
Comment (annetill) [reply src/test/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImplTest.java:121]: I don't know, why are they deleted?

[the remark this answers is not in the document]
Comment (geofjamg) [reply src/test/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImplTest.java:121]: No idea why it has been removed

[the remark this answers is not in the document]
Comment (EtienneLt) [reply src/test/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImplTest.java]: no idea either

**Pull request merged**